src/Symfony/Component/Form/Tests/AbstractTableLayoutTest.php

```diff
@@ -79,7 +79,7 @@ public function testRepeatedRow()
         /following-sibling::td
             [./input[@id="name_second"]]
     ]
-/following-sibling::tr[@style="display: none"]
+/following-sibling::tr[@style="display: none;"]
```

Comment:
can't we get rid of this change? I fear it might break cross testing - or do we need to bump `$supportedFeatureSetVersion` in child classes? @xabbuh wdyt?

Reply:
Those changes are only necessary because I chose to prepend the `display: none` in https://github.com/symfony/symfony/pull/33573/files#diff-c66befd9821a00f467f17bc882749b85R30 to avoid issue if the added "style" by the end user  doesn't end with a `;`. I guess we can find a work around to append it instead and revert all impacted files.

Reply:
I did it, look at https://github.com/symfony/symfony/pull/33573/files#diff-c66befd9821a00f467f17bc882749b85R30.